Classes/Emogrifier.php

```diff
@@ -270,9 +282,18 @@ public function emogrify()
             if ($nodesMatchingCssSelectors === false) {
                 continue;
             }
-
+            
             /** @var \DOMElement $node */
             foreach ($nodesMatchingCssSelectors as $node) {
+                // Skip excluded nodes
+                if (!empty($excludedNodes)) {
```

Comment:
We don't need the !empty check here - a foreach on an empty array is just a noop.